Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -1730,6 +1803,21 @@ void SourceBuffer::sourceBufferPrivateDidReceiveSample(MediaSample& sample)
     // duration set to the maximum of the current duration and the group end timestamp.
     if (m_groupEndTimestamp > m_source->duration())
         m_source->setDurationInternal(m_groupEndTimestamp);
+
+    // To avoid playback pipeline starvation start providing media data as soon as we can
+    const AtomicString& trackID = sample.trackID();
+    auto it = m_trackBufferMap.find(trackID);
```

Comment:
We probably want to auto& this.

Reply:
The iterator or the trackId? Iterator doesn't make sense to me.